src/__integ__/__snapshots__/themes.test.ts.snap

```diff
@@ -526,6 +526,7 @@ Object {
   "space-alert-message-right": "0px",
   "space-alert-vertical": "8px",
   "space-button-focus-outline-gutter": "3px",
+  "space-button-horizontal": "20px",
```

Comment:
Looks like VR compact mode is not covered here, and I think it would be easy to add it by adding something like `['compact', '#light/?visualRefresh=true&density=compact']` to [this array](https://github.com/cloudscape-design/components/blob/main/src/__integ__/themes.test.ts#L39). WDYT?

Reply:
Sure! Added and updated snapshots